### PR TITLE
Added Aarogya-Bot project by arghyadeep99

### PR DESCRIPTION
**Pull request by arghyadeep99**

### CODE REVIEW

File: projects/arghyadeep99/Aarogya-Bot.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+description: Aarogya Bot is an AI-driven chatbot that will help you answer your basic medical queries, amidst the Covid-19 pandemic, where moving out to consult a doctor isn't a luxury anymore. The chatbot can respond to your medical queries only to the best of its knowledge graph base.
+languages:
+  - Python
+  - JavaScript
+  - NodeJS
+  - VueJS
+  - HTML / CSS
+tags: # A list of keywords relevant to the contributors for this project
+  - Chatbot
+  - Healthcare
```
Comment (ramonh) [inline projects/arghyadeep99/Aarogya-Bot.mdx:13]:
```suggestion
  - Health
```

Let's please use existing tags! :)

File: projects/arghyadeep99/Aarogya-Bot.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - NLP
```
Comment (ramonh) [inline projects/arghyadeep99/Aarogya-Bot.mdx:14]:
```suggestion
  - Machine Learning
```

I'd love your opinion on this: We currently have an existing Machine Learning tag. Would NLP fit under this? What do you think?

Comment (arghyadeep99) [reply projects/arghyadeep99/Aarogya-Bot.mdx:14]: To be honest, today NLP does have its own "tag" for itself, than just being declared Machine Learning, but then, we will also have RL, CV, GANs, etc. So, if the requirement is to not increase the tags, we can keep it as Machine Learning tag!

Comment (ramonh) [reply projects/arghyadeep99/Aarogya-Bot.mdx]: Thank you so much for weighing in! 

We definitely don't want to overload potential contributors with too many tags, so I agree, let's keep it broad. Thank you!

**Review by ramonh — APPROVED** [review]

**Comment by ramonh** [comment]
Thank you so much, @arghyadeep99! Welcome to Port 🚢

**Pull request merged**